### PR TITLE
Docs(web-react, web-twig) Add FileUploader demo #DS-896

### PR DESCRIPTION
**Pull request by pavelklibani**
## Description

FileUploader demo in web-react is now same as demo in web and web-twig.

#### Minor fixes:
- Icon color in `web-twig` was not changing correctly in the disabled state (chrome browser).
image

- Drag&Drop in `web-react` was still working when the field was disabled.
image

### Issue reference

[Demo: FileUploader má ve všech technologiích stejná dema](https://jira.lmc.cz/browse/DS-896)

### CODE REVIEW

File: packages/web-react/src/components/FileUploader/utils.ts [hunk 1]
```diff
@@ -48,4 +48,15 @@ const image2Base64Preview = (file: File, maxWidth: number, callback: (base64Prev
   reader.readAsDataURL(file);
 };
 
-export { getAttachmentInput, image2Base64Preview };
+const base64ToByteArray = (base64Image: string) => {
```
Comment (literat) [inline packages/web-react/src/components/FileUploader/utils.ts:51]: Can we add a test to this file, please?

**Review by literat — CHANGES_REQUESTED** [review]
Please, move the base64 demo image to a separate file location and add the test for the utility function. Thanks.

**Review by literat — APPROVED** [review]
LGTM 👍

File: packages/web-react/src/components/FileUploader/constants.ts [hunk 2]
```diff
@@ -9,3 +9,6 @@ export const DEFAULT_ERROR_MESSAGE_UNSUPPORTED_FILE = 'This file type is not sup
 export const DEFAULT_ICON_NAME = 'file';
 export const DEFAULT_BUTTON_LABEL = 'Remove';
 export const DEFAULT_EDIT_BUTTON_LABEL = 'Edit';
+
+export const DEMO_ATTACHMENT_BASE64_IMAGE =
```
[the remark this answers is not in the document]
Comment (pavelklibani) [reply packages/web-react/src/components/FileUploader/constants.ts]: moved to different file inside demo folder

File: packages/web-react/docs/DocsSections.tsx [hunk 3]
```diff
@@ -1,15 +1,28 @@
 import React, { ReactNode } from 'react';
+import { Tag } from '../src/components';
 
 interface DocsSectionProps {
   children: ReactNode;
   title: string;
+  tag?: string;
```
Comment (crishpeen) [inline packages/web-react/docs/DocsSections.tsx:7]: 🔡 😬

File: packages/web-react/src/components/FileUploader/demo/FileUploaderInputMultipleWithFileQueueBehaviorControl.tsx [hunk 4]
```diff
@@ -15,37 +11,37 @@ const Story = (args: FileUploaderBaseProps) => {
...
-        validationText="Validation message"
-        maxUploadedFiles={3}
+        linkText="Upload your file(s)"
+        maxUploadedFiles={2}
+        name="attachments"
         /* eslint-disable-next-line no-console */
         onError={(error) => console.error('My error log', error)}
         queueLimitBehavior="hide"
-        isMultiple
+        validationText="Validation message"
```
Comment (crishpeen) [inline packages/web-react/src/components/FileUploader/demo/FileUploaderInputMultipleWithFileQueueBehaviorControl.tsx:35]: by the way does it make sense to keep validationText here if it doesn't work without validationstate? Especially in only visual demos.

File: packages/web-react/src/components/FileUploader/utils.ts
Comment (crishpeen) [inline packages/web-react/src/components/FileUploader/utils.ts:1]: where did I already see this file...? :D

**Review by crishpeen — APPROVED** [review]

File: packages/web-react/src/components/FileUploader/demo/FileUploaderInputMultipleWithFileQueueBehaviorControl.tsx (hunk 4 above)
Comment (pavelklibani) [reply packages/web-react/src/components/FileUploader/demo/FileUploaderInputMultipleWithFileQueueBehaviorControl.tsx:35]: probably not, i will remove it

File: packages/web-react/docs/DocsSections.tsx (hunk 3 above)
Comment (pavelklibani) [reply packages/web-react/docs/DocsSections.tsx:7]: sorted

File: packages/web-react/src/components/FileUploader/utils.ts
Comment (pavelklibani) [reply packages/web-react/src/components/FileUploader/utils.ts:1]: 😁

**Pull request merged**